drivers/virtualbox/virtualbox.go

```diff
@@ -177,17 +177,14 @@ func (d *Driver) Create() error {
 		if err := os.Mkdir(imgPath, 0700); err != nil {
 			return err
 		}
-
 	}
 
 	if d.Boot2DockerURL != "" {
 		isoURL = d.Boot2DockerURL
 		log.Infof("Downloading %s from %s...", isoFilename, isoURL)
-		if err := b2dutils.DownloadISO(commonIsoPath, isoFilename, isoURL); err != nil {
+		if err := b2dutils.DownloadISO(imgPath, isoFilename, isoURL); err != nil {
```

Comment:
this will change the cached boot2docker.iso file which means I need to cleanup cached boot2docker.iso manually if it doesn't work before performing command `docker-machine create` without `--virtualbox-boot2docker-url`. shall we use `storePath` in this context to avoid caching _specific_ iso file when creating docker machine with specific iso file download URL?